tests/test_vms_platform.py

```diff
@@ -155,25 +139,18 @@ def _ensure_keyring_package_exists_and_has_correct_key(self, vm):
 
     def _ensure_trusted_keyring_securedrop_key_removed(self, vm):
         """
-        Ensures the production key is no longer found in the default apt keyring
-        In testeing dev/staging, that keyring will be used for the test apt key,
-        the goal is to reduce of the production key
+        Ensures the production key is no longer found in the default apt keyring.
+        In dev/staging environments, that keyring will be used for the test apt key,
+        so we only check against the production fingerprint. The goal is to ensure
+        the production key is kept separate from other keys.
         """
         # apt-key finger doesnt work here due to stdout/terminal
         cmd = "gpg --no-default-keyring --keyring /etc/apt/trusted.gpg -k"
         stdout, stderr = vm.run(cmd)
         results = stdout.rstrip().decode("utf-8")
-        fpf_gpg_pub_key_fp = "22245C81E3BAEB4138B36061310F561200F4AD77"
+        fpf_gpg_pub_key_fp = "2359E6538C0613E652955E6C188EDD3B7B22E6A3"
```

Comment:
This is why https://github.com/freedomofpress/securedrop-debian-packaging/issues/367 was not caught.

Reply:
right, because it's the new key that'll be in the keyring